admin/common_auth/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url(r'^login/$', views.login, name='login'),
```

Comment:
If you change this to `r'^login/?$'` it will not require the trailing slash, same goes for the next line. The ? makes the previous character optional.